fleetdm/fleet
# Request review automatically for non-codeowners

https://github.com/fleetdm/confidential/issues/3146

To test this works:
- sam should be able to change ceo handbook and be autoapproved
- mike submitting a PR to the ceo handbook AND `tools/fleetctl-npm/README.md` at the same time should request review from Sam but auto-approve for mike
- Jarod's use case of creating an article should now work, even if his PR contains images

.
